.github/workflows/dev_builds.yaml

```diff
@@ -0,0 +1,69 @@
+name: Dev builds
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      OPERATOR_IMAGE_NAME: sumologic-kubernetes-collection-helm-operator-dev
+      BUNDLE_IMAGE_NAME: sumologic-kubernetes-collection-helm-operator-bundle-dev
+
+      ECR_URL: public.ecr.aws/u5z5f8z6
+      OPENSOURCE_ECR_URL:  public.ecr.aws/a4t4y2n3
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Extract tag
+        id: extract_tag
+        run: echo "::set-output name=tag::$(echo $(git describe --tags --always))"
+
+      - name: Print tag
+        run: echo "Running release build for ${{ steps.extract_tag.outputs.tag }}"
+
+      - name: Extract release number
+        id: extract_release_number
+        # count git tags omitting rc versions
+        run: echo "::set-output name=release_number::$(echo $(git tag | sed '/rc/d' | wc -l))" 
+
+      - name: Login to AWS public ECR
+        run: make login
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_PUBLIC_ECR_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_PUBLIC_ECR_SECRET_ACCESS_KEY }}
+
+      - name: Build and push operator image to public ECR
+        run: make docker-build docker-push \
+          IMG=$(ECR_URL)/$(OPERATOR_IMAGE_NAME):${{ steps.extract_tag.outputs.tag }} \
+          VERSION=${{ steps.extract_tag.outputs.tag }} \
+          RELEASE_NUMBER=${{ steps.extract_release_number.outputs.release_number }}
+
+      - name: Build and push bundle image to public ECR
+        run: make bundle-build bundle-push \
+          BUNDLE_IMG=$(ECR_URL)/$(BUNDLE_IMAGE_NAME):${{ steps.extract_tag.outputs.tag }}
+
+      - name: Login to Open Source ECR to publish operator
+        run: make login-opensource
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ECR_OPERATOR_ID_DEV }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.ECR_OPERATOR_KEY_DEV }}
```

Comment:
Where do those names come from? Why not use the open source prefix as we used elsewhere to keep consistent?